### DISCUSSION TITLE
How to distinguish projects with multiple windows open?

### DISCUSSION THREAD

**Discussion by DarkSymm** [zed-industries/zed#13192] [Feature Requests]
Zed uses a project's folder name in the top window toolbar — but if I have multiple windows open and they all have the happen to have the same project folder name I don't see anyway to easily distinguish which project is which.

Is there anyway to include the full file path or specify a project name?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
